src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1783,6 +1794,23 @@ private static bool IsBlank(FormulaValue arg)
             return false;
         }
 
+        private static FormulaValue IsEmpty(IRContext irContext, FormulaValue[] args)
+        {
+            if (args[0] is BlankValue)
+            {
+                return new BooleanValue(irContext, false);
+            }
+
+            if (args[0] is TableValue tv)
+            {
+                return new BooleanValue(irContext, tv.Count() == 0);
```

Comment:
>tv.Count() [](http://example.com/codeflow?start=51&length=10)

We could have an TableValue.IsEmpty property that calls Row.Any() in case the table has millions of records.

Reply:
Good point, changed to `!tv.Rows.Any()` so that we don't need to count the number of rows.